lambdas/retrieve-cri-credential/src/main/java/uk/gov/di/ipv/core/retrievecricredential/RetrieveCriCredentialHandler.java

```diff
@@ -96,14 +95,24 @@ public RetrieveCriCredentialHandler() {
     public APIGatewayProxyResponseEvent handleRequest(
             APIGatewayProxyRequestEvent input, Context context) {
         LogHelper.attachComponentIdToLogs();
-        IpvSessionItem ipvSessionItem = null;
-
-        RetrieveCriRequestDto request =
-                RequestHelper.convertRequest(input, RetrieveCriRequestDto.class);
 
+        IpvSessionItem ipvSessionItem;
         try {
             String ipvSessionId = RequestHelper.getIpvSessionId(input);
             ipvSessionItem = ipvSessionService.getIpvSession(ipvSessionId);
+        } catch (HttpResponseExceptionWithErrorBody e) {
+            ErrorResponse errorResponse = e.getErrorResponse();
+            LogHelper.logOauthError(
```

Comment:
This wouldn't be an oauth error here. Not sure what the result of the logs would be using the logOauthError if its going to make it look like an oauth error and could be a bit confusing in the logs etc? Might be better to swap to a normal error log here